homeassistant/components/media_player/__init__.py

```diff
@@ -452,6 +452,11 @@ def media_image_url(self):
         return None
 
     @property
+    def media_image(self):
+        """Binary image, content type and hash of playing media."""
+        return (None, None, None)
```

Comment:
Need only `(None, None)` since it will evertime create a hash on https://github.com/postlund/home-assistant/blob/654ba940dc8d93e02445bf8fcc2255895a90472c/homeassistant/components/media_player/__init__.py#L826-L828 so we have no improve with that.